dags/taar_weekly.py

```diff
@@ -43,14 +45,13 @@
     spot_bid_price_percent=100,
     max_instance_count=60,
     enable_autoscale=True,
-    start_date='20190527',
-    pypi_libs=['mozilla-taar3==0.4.5', 'mozilla-srgutil==0.1.10', 'python-decouple==3.1'],
-    env=mozetl_envvar(
-        "taar_ensemble",
-        {
-            "date": "{{ ds_nodash }}",
-        },
-    ),
+    start_date=datetime(2019, 7, 27),
```

Comment:
Is this date really necessary, since it's provided as a default argument via the DAG.

Reply:
I've dropped this parameter in my latest patch.